### PR TITLE
Fix skid re-render

### PR DESCRIPTION
**Pull request by bjornstar**
- [Skid] No longer re-renders every time anything in the store changed
- [Skid] Rotates the plane geometry back when unloading
- [Boost] No longer re-renders every time a control changed
- [Store] Moved `chassisBody` up to `api` as there's nothing else inside api
- [Help] Renamed `.controls` to `.help` to match the component name
- [Leaderboard] Took out an unnecessary div layer, renamed css from `.controls` to `.leaderboard`
- [CSS] Separated `.popup` from `.help`

### CODE REVIEW

File: src/effects/Skid.tsx [hunk 1]
```diff
@@ -20,21 +19,26 @@ interface SkidProps {
...
       if (index === count) index = 0
     }
   })
 
-  useLayoutEffect(() => void ref.current!.geometry.rotateX(-Math.PI / 2), [])
+  useLayoutEffect(() => {
+    ref.current!.geometry.rotateX(-Math.PI / 2)
+    return () => {
+      ref.current!.geometry.rotateX(Math.PI / 2)
+    }
```
Comment (njm222) [inline src/effects/Skid.tsx:40]: why is this needed?

**Review by njm222 — APPROVED** [review]

File: src/effects/Skid.tsx (hunk 1 above)
Comment (bjornstar) [reply src/effects/Skid.tsx:40]: Every time the component renders the layoutEffect call rotates the geometry, so every render it rotates an additional 90 degrees. This rotates it back when the component unmounts so that it's always the correct rotation.

It would be nice to be able to set the rotation in the declaration of the geometry or being able to set the rotation directly, but I couldn't see a way to do that.

Comment (njm222) [reply src/effects/Skid.tsx:40]: Ahhh thanks for the explanation!

Can't you pass rotate as a prop to instanced mesh? or to planeGeometry?
Or does that not work for instanced meshes?

Comment (bjornstar) [reply src/effects/Skid.tsx:40]: I couldn't see a way to do that.

Comment (njm222) [reply src/effects/Skid.tsx:40]: like this:

```jsx
<planeGeometry rotation={[x,y,z]} />
```

https://docs.pmnd.rs/react-three-fiber/API/objects
